### ISSUE TITLE
postcss-color-function support

### ISSUE THREAD

**Issue by max** [salsify/ember-css-modules#23]
Currently it isn't possible to use an imported `@value` in a PostCSS color function. For example:

```css
@value --light-gray from "things/styles/colors";

.foobar {
  background: color(--light-gray alpha(50%));
}
```

This results in:

```
Unable to parse color from string "i__const___light_gray_473"
```

By comparison this works fine:

```css
@value --light-gray: #999;

.foobar {
  background: color(--light-gray alpha(50%));
}
```

I also tried moving `postcss-color-function` to the before hook but then the error message is:

```
Unable to parse color from string "--light-gray"
```

Which makes sense given that at that stage it's just a string.

**Comment by max** [comment]
Looks like this might be blocked by https://github.com/css-modules/postcss-modules-values/issues/5.

**Comment by dfreeman** [comment]
I get the distinct impression you're the first person to use this who already has a preferred suite of PostCSS plugins :)

I probably won't get a chance to dig in on this one for a few days, but the issue you linked looks like a good starting point. I may need a new issue label for "stuff I'm pretty sure is the underlying loader's fault but haven't actually tracked down yet."

**Comment by dfreeman** [comment]
Just released 0.3.2, which includes a broccoli-css-modules version bump that should take care of this.

**Comment by NullVoxPopuli** [comment]
> I get the distinct impression you're the first person to use this who already has a preferred suite of PostCSS plugins :)

count me as #2, I guess.
I'm having a hard time getting my previously working postcss config working with css-modules (only adopted for component-co-location support)
